docs/root/intro/version_history.rst

```diff
@@ -46,6 +46,10 @@ Version history
 * mysql: added a MySQL proxy filter that is capable of parsing SQL queries over MySQL wire protocol. Refer to ::ref:`MySQL proxy<config_network_filters_mysql_proxy>` for more details.
 * http: added :ref:`max request headers size <envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.max_request_headers_kb>`. The default behaviour is unchanged.
 * http: added modifyDecodingBuffer/modifyEncodingBuffer to allow modifying the buffered request/response data.
+* http: added encodeComplete/decodeComplete. These are invoked at the end of the stream, after all data has been encoded/decoded respectively. Default implementation is a no-op.
```

Comment:
Looks like the other lines are irrelevant, can you merge master?

Also start the jwt line with `jwt_authn: `, and sort please.

Reply:
BTW, line 82 has `tracing:` in the midst of a few `upstream:` lines.

Reply:
ah thanks, we will fix that during the release process.